### DISCUSSION TITLE
Quiz P4DS dan PS

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#687] [Q&A] [answered]
Di nomor 6 disuruh uji hipotesis menggunakan Z test. Di kelas PS kemarin dikatakan uji hipotesis menggunakan Z test hanya bila n>30, sementara di soal n=1.

**Comment by kinantytasyao** [comment, marked as answer]
Dear @ArdiantheGreat 

Hal tersebut karena pada soal syarat utama penggunaan uji Z bisa dipenuhi, yaitu sudah diketahui standar deviasinya

Demikian yang bisa kami sampaikan. Semoga membantu :)
